### DISCUSSION TITLE
Scale to zero?

### DISCUSSION THREAD

**Discussion by rmortes** [coollabsio/coolify#4516] [Feature Requests]
Is there a way to scale to zero a resource? If not I think it'd be a pretty interesting feature for people with small homelabs that still want to host a lotta services that won't get used often.

**Comment by RayBB** [comment]
@rmortes ever figure out a way to accomplish this?

**Reply by RayBB** [reply]
Ah there was some comment by @peaklabs-dev about it in Oct 2025 saying it's being considered but not certain.
https://github.com/coollabsio/coolify/issues/5685#issuecomment-3369029209

Personally, I think it's a great idea because can make it so people can literally run 5x the apps if not more.

For example, I use karakeep, the read it later app. Maybe a few times per day I drop a url in and then once a week I sit down and read the articles stored. That sucker takes 600+ MB of ram all the time. Well if it was only 600mb for like the few hours per week then I could more easily have other services taking up that space.

Something crude like shut down after 30 minutes of no traffic and then auto start is pretty easy to do with a sidecar proxy but it's really not fun to manage that manually.
